pkgs/by-name/fr/freeshow/package.nix

```
@@ -0,0 +1,78 @@
+{
+  lib,
+  fetchurl,
+  stdenv,
+  pkgs,
+  appimageTools,
+}:
+
+let
+  pname = "freeshow";
+  version = "1.2.1";
+  src =
+    fetchurl
+      {
+        x86_64-darwin = {
+          url = "https://github.com/ChurchApps/FreeShow/releases/download/v${version}/FreeShow-${version}.dmg";
+          hash = "sha256-OO0uQ6oS1GKiBGz3Qt9jDCY+qdWlgTOY+SiLJB5xQ3c=";
+        };
+        x86_64-linux = {
+          url = "https://github.com/ChurchApps/FreeShow/releases/download/v${version}/FreeShow-${version}.AppImage";
+          hash = "sha256-RBI8IkxY6Xd36vCVDJy9sqpEisB/48hwQo+mg5XTCOs=";
+        };
+      }
+      .${stdenv.system} or (throw "Unsupported system: ${stdenv.system}");
+
+  appimageContents = appimageTools.extract { inherit pname version src; };
+
+  meta = {
+    homepage = "https://github.com/ChurchApps/FreeShow";
+    description = "A free and open-source, user-friendly presenter software. ";
```

```suggestion
    description = "Free and open-source, user-friendly presenter software";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes